### ISSUE TITLE
Loss of note names, octave number and block length on long lines

### ISSUE THREAD

**Issue by angs** [SarutaSan72/Yass#138]
An error that happens on high bpm songs with long lines - a line with a length of at least 102 beats will not show note names etc (A#4 3).

This is happening since 2.4.2 because of line 3745 in YassSheet.java: that says `if (wSize > 8) {`. 

The longest line I have at hand is 197 beats and had to go as low as (wSize > 3) to make this one song work, but better might be adding some kind of logic for always allowing showing on single lines. As pseudocode: `if (wSize > 8 || isSingleLine)`.

**Comment by SarutaSan72** [comment]
are there really lines that have more than 100 beats?
better 

- break such lines for better UX
- or add auto-correction

**Comment by angs** [comment]
I think what I meant was quarter beats, or whatever the smallest possible units of time in YASS are. 

I ran the numbers on the dataset of all the songs I had at hand and the results are: ~4% of lines are at least 102 units long. The longest I found was 373 (quarter)beats, which was the last line in Iron Maiden's The Clairvoyant, "And be reborn again", which sustains for 14 seconds. The bpm could possible have been lower, but even just the last word lasts for 12.3 seconds. At a conservative bpm of 300 this would be 153 units long, so any splitting wouldn't have helped. With a bpm of 400 the limit of hitting the bug is a 3.825 seconds per line, which can easily be reached with a single word.
